client-events.js

```diff
@@ -0,0 +1,71 @@
+console.warn("\n1. Make sure server is running i.e. `npm run server`"+
+"\n2.Run two or more instances of client-events, one for sending and others for receiving client events")
+console.warn("\nOnly compatible with unix/bash shell, git bash can be used in windows")
+
+const Pusher = require('pusher-js');
+
+const pusherRealtime = new Pusher('appid.keyid', {  // replace with first part of API key before :
+    wsHost: 'realtime-pusher.ably.io',
+    wsPort: 443,
+    disableStats: false,
```

Comment:
shouldn't `disableStats` be `true`? https://faqs.ably.com/using-the-ably-pusher-protocol-adapter

Reply:
Actually, I am not sure what it does ha .... but when I tested it, everything seem to work. Is it supposed to be removed?